packages/runtime-dom/types/jsx.d.ts

```diff
@@ -249,7 +249,7 @@ export interface HTMLAttributes extends AriaAttributes, EventHandlers<Events> {
   innerHTML?: string
 
   class?: any
-  style?: string | CSSProperties
+  style?: string | CSSProperties | Array<string | CSSProperties>
```

Comment:
This array can be deeply nested.

Reply:
@znck WDYT?

Reply:
Do we support deeply nested styles? Same issue is popping for `class` attribute.

Reply:
Yes: https://codepen.io/kaelwd/pen/KKNmvqY?editors=1010

Reply:
> Same issue is popping for class attribute

wdym? `class` flattens arrays too but is already typed as `any` so isn't a problem.